README.md

```diff
@@ -51,6 +95,13 @@ flux create source git addons \
     --branch=main # This should be replaced with your branch for testing your changes
 ```
 
+This creates a flux GitRepository resource. This is basically a monitoring service that looks for changes in the given branch in the repository and then syncs those changes. Since this project uses git we create GitRepository resource.
```

Comment:
Flux is not a monitoring service. Flux keeps Kubernetes clusters in sync with configuration sources such as Git repositories and automate configuration updates when available. Please update the verbiage accordingly pls

Reply:
Well flux keeps things in sync by monitoring the configured repository, it actually polls the repo every minute. But I can understand where the overload of "monitoring" as in "systems monitoring" and "monitoring" as in watching anything" can be misleading.